popupModal.css

```diff
@@ -0,0 +1,433 @@
+  color: #344563;
+}
+
+.projectJob {
+  display: flex;
+  margin-left: -60px;
+  list-style-type: none;
+}
+
+.jobInformation {
+  font-weight: 600;
+  margin-right: 10px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-fulldesc {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  margin-left: -10px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.jobInformation::before {
+  content: url('./assets/desktop/Counter.png');
+  margin-right: 10px;
+}
+
+.project_tags_container {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  margin: 16px 0 29px 30px;
+  width: 100%;
+}
+
+.btn_containers {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  margin: -19px 0 29px 4px;
+}
+
+.project_tag {
+  padding: 7px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 16px;
+  margin-right: 10px;
+  line-height: 16px;
+  color: #6070ff;
+  background: #ebebff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.view_project_btn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 124px;
+  height: 48px;
+  border-radius: 8px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  background-color: #fff;
+  margin-left: -10px;
+  transition: all 0.5s ease-in-out;
+}
+
+.view_project_btn::after {
+  content: '';
+  background: #6070ff;
+  display: block;
+  position: absolute;
+  padding: 20px;
+  margin-left: -20px;
+  margin-top: -12%;
+  opacity: 0;
+  transition: all 0.8s;
+  border-radius: 100%;
+}
+
+.view_project_btn:active::after {
+  padding: 0;
+  margin: 0;
+  opacity: 1;
+  transition: 0s;
+}
+
+.view_project_btn:hover {
+  background-color: #6070ff;
+  color: white;
+  transform: scale(1.1);
+}
+
+.view_project_btn:active {
+  background-color: #172b4d;
+  color: white;
+}
+
+.closed {
+  display: none !important;
+}
+
+.resume_btn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 170px;
+  height: 48px;
+  border-radius: 8px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  background-color: #fff;
+  margin-left: 50px;
+}
+
+.resume_btn::after {
+  content: '';
+  background: #6070ff;
+  display: block;
+  position: absolute;
+  padding: 20px;
+  margin-left: -20px;
+  margin-top: -12%;
+  opacity: 0;
+  transition: all 0.8s;
+  border-radius: 100%;
+}
+
+.resume_btn:active::after {
+  padding: 0;
+  margin: 0;
+  opacity: 1;
+  transition: 0s;
+}
+
+.resume_btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.resume_btn:active {
+  background-color: #172b4d;
+  color: white;
+}
+
+.tags_flex {
+  display: flex;
+  margin-left: -40px;
+}
+
+.btn_child_flex {
+  display: flex;
+  margin-left: -40px;
+}
```

Comment:
Kindly please remove the ">" symbol that is appearing next to the buttons in your pop-up window. This should not be part of the project. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/202856689-b731c483-ced3-488a-9ce7-079386177d68.png)